third_party/3/typed_ast/ast35.pyi

```diff
@@ -89,7 +89,7 @@ class Delete(stmt):
 class Assign(stmt):
     targets = ...  # type: typing.List[expr]
     value = ...  # type: expr
-    type_comment = ...  # type: Optional[str]
+    type_comment = ...  # type: Union[AST, Optional[str]]
```

Comment:
@gvanrossum It's OK without typed_ast, but mypy will complain about its own code
```
mypy/fastparse.py:411: error: Argument 1 to "parse_type_comment" has incompatible type
"Union[AST, str]"; expected "str"
```
So that this PR should go together with the one at mypy. The PR to mypy is made in such a way that it will continue to work with old typed_ast.